### ISSUE TITLE
Accessibility - HTML Errors

### ISSUE THREAD

**Issue by MalcolmShazell** [epimorphics/ukhpi#155]
This error is repeated a number of times - Ian Git appears to be messing up the HTML you could use to identify the problem - is emailing it to you easier or can you spot the problem with the line numbers?

Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 6594; to line 1, column 6645
    s'><label><input checked readonly type='checkbox' value='all'><a hre

4.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 6695; to line 1, column 6738
    el><label><input readonly type='checkbox' value='det'><a hre

5.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 6814; to line 1, column 6857
    el><label><input readonly type='checkbox' value='sem'><a hre

6.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 6938; to line 1, column 6981
    el><label><input readonly type='checkbox' value='ter'><a hre

7.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 7057; to line 1, column 7100
    el><label><input readonly type='checkbox' value='fla'><a hre

8.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 10887; to line 1, column 10938
    s'><label><input checked readonly type='checkbox' value='all'><a hre

9.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 10988; to line 1, column 11031
    el><label><input readonly type='checkbox' value='det'><a hre

10.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 11107; to line 1, column 11150
    el><label><input readonly type='checkbox' value='sem'><a hre

11.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 11231; to line 1, column 11274
    el><label><input readonly type='checkbox' value='ter'><a hre

12.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 11350; to line 1, column 11393
    el><label><input readonly type='checkbox' value='fla'><a hre

13.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 15165; to line 1, column 15216
    s'><label><input checked readonly type='checkbox' value='all'><a hre

14.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 15266; to line 1, column 15309
    el><label><input readonly type='checkbox' value='det'><a hre

15.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 15385; to line 1, column 15428
    el><label><input readonly type='checkbox' value='sem'><a hre

16.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 15509; to line 1, column 15552
    el><label><input readonly type='checkbox' value='ter'><a hre

17.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 15628; to line 1, column 15671
    el><label><input readonly type='checkbox' value='fla'><a hre

18.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 19433; to line 1, column 19484
    s'><label><input checked readonly type='checkbox' value='all'><a hre

19.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 19534; to line 1, column 19577
    el><label><input readonly type='checkbox' value='det'><a hre

20.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 19653; to line 1, column 19696
    el><label><input readonly type='checkbox' value='sem'><a hre

21.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 19777; to line 1, column 19820
    el><label><input readonly type='checkbox' value='ter'><a hre

22.Error: Attribute readonly is only allowed when the input type is date, datetime-local, email, month, number, password, search, tel, text, time, url, or week.

    From line 1, column 19896; to line 1, column 19939
    el><label><input readonly type='checkbox' value='fla'><a hre

**Comment by ijdickinson** [comment]
Should be fixed now.

**Comment by MalcolmShazell** [comment]
confirm fixed
